index.html

```diff
@@ -27,14 +30,14 @@
       </nav>
 
 
-      <section>
-        <h1>Startrly</h1>
-        <sub>A catchy tagline for a great solution</sub>
-        <a href="#">Some Call to Action</a>
-      </section>
+      <header class="header-box">
+        <div class="header-startrly"><h1>Startrly</h1></div>
+        <div class="header-tagline"><sub>A catchy tagline for a great solution</sub></div>
+        <div><a href="#" class="header-button">Some Call to Action</a></div>
+      </header>
```

Comment:
i liked how you wrapped these three elments  in div! but why not add class directly in h1 and sub? (asking bcz i don't know hehe)